src/Components/MainMenu/MenuItem/MenuItem.js

```diff
@@ -40,17 +40,47 @@ const propTypes = {
    * start with the item expanded
    */
   startExpanded: PropTypes.bool,
+  /**
+   * Current location, used for active item detection
+   */
+  location: PropTypes.exact({
+    hash: PropTypes.string,
+    pathname: PropTypes.string,
+    search: PropTypes.string,
+    state: PropTypes.string,
+  }),
+  nested: PropTypes.bool,
 };
 
 const defaultProps = {
   containsActiveItem: false,
   startExpanded: false,
+  location: {},
+  nested: false,
```

Comment:
why is this is prop types? Should just be an instance property?

Reply:
It's passed in from MainMenue where it's checking the whole object